src/app.tsx

```diff
@@ -118,7 +119,13 @@ export function App(props: AppProps) {
   KeyboardShortcuts()
 
   return (
-    <div id="parts-kit">
+    <div className={cx(
+      'font-sans',
+      {
+        'light': themeStore.mode === 'light',
+        'dark': themeStore.mode === 'dark',
+      }
+    )}>
```

Comment:
Moving into the shadow DOM made it necessary to move theming into the React app. 

I created a new store that recreates & centralizes this behavior.